app/api_topologies.go

```diff
@@ -196,6 +195,17 @@ func MakeRegistry() *Registry {
 		},
 	}
 
+	k8sControllersTypeFilter := APITopologyOptionGroup{
+		ID:         "grouptype",
+		Default:    "",
```

Comment:
When I test this branch, the type filter has nothing selected. Unlike, say, the namespace filter, which has "All Namespaces" selected.

I suspect that's because the above ought to be `Default: "none"`. Even though you say this branch fixes #2525. (in which case, btw, I would expect existing filters, such as the namespace filter, to make use of this functionality - which suggests this change would be better off in its own PR)